pages/common/fls.md

```diff
@@ -0,0 +1,16 @@
+# fls
+
+> List files and directories in an image file or device.
+> More information: <https://wiki.sleuthkit.org/index.php?title=Fls>.
+
+- Build a recursive fls list over a device, output pathes will start with C:
+
+`fls -r -m {{C:}} {{/dev/loop1p1}}`
+
+- Analyse a single partition, providing the sector offset at which the file system starts in the image:
+
+`fls -r -m {{C:}} -o {{sector}} {{path/to/imagefile}}`
```

Comment:
Since you are using the word file, I would suggest to use image_file because we use snake_case convention.